templates/taxi/car_detail.html

```diff
@@ -14,9 +14,23 @@ <h1>
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
   <h1>Drivers</h1>
   <hr>
+  
   <ul>
     {% for driver in car.drivers.all %}
+
       <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
     {% endfor %}
+  {% if user in car.drivers.all %}
+        <form method="post">
+            {% csrf_token %}
+            <button type="submit" name="remove" class="btn btn-danger">Remove me from this car</button>
```

Comment:
you use here backward relationship but did you fetched it from db? because otherwise here will be N+1 issue